boot/zephyr/Kconfig

```diff
@@ -14,6 +14,27 @@ config MCUBOOT
 	select MPU_ALLOW_FLASH_WRITE if ARM_MPU
 	select USE_CODE_PARTITION if HAS_FLASH_LOAD_OFFSET
 
+config PARTITION_MANAGER_RESERVED_SPACE_MCUBOOT
```

Comment:
Man, I wish these macros where shorter :) Somewhere else I have seen Partition Manager macros prefixed with just `PM`, that's maybe a bit cryptic; something in-between would be better (just on the top of my head: `PARTMAN` ??). 

`RESERVED_SPACE` is informative but `AREA` would also work and is the same terminology used in `flash_map` and Device Tree. Otherwise it might make sense to stick to your own terminology and call that a `PARTITION_SIZE`, same length.

In any case the macro prefix should be aligned.

Reply:
`PARTMAN` sounds awesome. A small comment could be enough to make it clear even with `PM` though.

Reply:
Will change to `PM_PARTITION_SIZE_x` and add comments in the kconfig files